mantis-common/src/main/java/io/mantisrx/common/metrics/Metrics.java

```diff
@@ -179,6 +180,7 @@ public Map<MetricId, Timer> timers() {
     }
 
     public static class Builder {
+        private static Iterable<Tag> CommonTags = Collections.emptyList();
```

Comment:
What's the rationale for having this singleton state, given that we already have the SpectatorRegistryFactory, which allows you to set a registry with a given set of common tags?

Reply:
Inside Springboot job runtime the registry is no longer built/control by mantis worker and there is no API right now to set common tags in the SBN registry injected.